/**:
  ros__parameters:
    joy_msg: "/joy"
    cmd_msg: "/fake_cmd"
    scan_msg: ""
    grid_msg: "/bev/occupancy_grid"
    max_linear_vel : 0.5
    max_side_vel : 0.0
    max_angular_vel : 30.0
    acc: 1.0
    dt: 0.05
    #
    collision_radius: 0.2
    collision_margin: 0.1
    resolution_x: 4
    resolution_y: 0
    resolution_th: 4
